VocabularyTrainer/AddWord/AddNewWordViewController.swift

```diff
@@ -83,6 +83,16 @@ final class AddNewWordViewController: UIViewController {
     private var translationHasText = false
     private var completed: (() -> Void)?
     
+    var hasDuplicates : Bool {
+        let word = newWordTextField.text ?? ""
+        guard let selectedLanguage = selectedLanguage,
+              let dictionary = UserDefaults.standard.dictionary(forKey: selectedLanguage) as? [String: String] else {
+                  return false
+              }
+        let matches = dictionary.map{String($0.key) }.filter({ $0.lowercased().elementsEqual(word.lowercased()) })
```

Comment:
Please split add some linebreaks
```suggestion
        let matches = dictionary
          .map { String($0.key) }
          .filter { $0.lowercased().elementsEqual(word.lowercased()) }
```